tooling.rb

```diff
@@ -188,7 +196,7 @@ def create_npk(src_dir, npk, manifest, arch_dir, pack_config)
     end.join(' ')
     # TODO: The compression algorithm should be target and not host specific!
     squashfs_comp = OS.linux? ? 'gzip' : 'zstd'
-    shell "mksquashfs #{tmp_root} #{fsimg} -all-root -comp #{squashfs_comp} -no-progress -info #{pseudofiles}"
+    shell "mksquashfs #{folder_to_package} #{fsimg} -all-root -comp #{squashfs_comp} -no-progress -info #{pseudofiles}"
```

Comment:
So we put all files inside a folder `/$VERSION/`? Does that make sense?
Shouldn't we keep them in the root of the filesystem and just have the squashfs mount target folder versioned?

Example:
- Resource container `resource` contains just `test.json`, versions `1.0.0` and `1.2.0` exist.
- Resource container `1.0.0` NPK filesystem content: `/test.json`
- Resource container `1.2.0` NPK filesystem content: `/test.json`
- North mounts `resource_v1.0.0.npk` to `<run_dir>/resource/1.0.0/` (file at `<run_dir>/resource/1.0.0/test.json`)
- North mounts `resource_v1.2.0.npk` to `<run_dir>/resource/1.2.0/` (file at `<run_dir>/resource/1.2.0/test.json`)
- Container A uses `1.0.0` of resource container at `/res`: North bind-mounts `<run_dir>/resource/1.0.0/` to `<container A root>/res`
- Container B uses `1.2.0` of resource container at `/res`: North bind-mounts `<run_dir>/resource/1.2.0/` to `<container B root>/res`

Reply:
in the original resource container we do not have a version folder, just the usual structure. So it always looks like this:
```
examples/container/resource/hello_message
├── manifest.yaml
└── root
    ├── hello
    └── pad
```
if we increase the version, the version field in the manifest is changed along with all the changes the new version brings.
but if we put it into the squashfs image, it looks like this:
```
squashFS-image: fs.img
Parallel unsquashfs: Using 2 processors
2 inodes (41 blocks) to write

squashfs-root
squashfs-root/0.1.2
squashfs-root/0.1.2/hello
squashfs-root/0.1.2/pad
```
does that make sense?